src/ng-xform/fields/custom-field.ts

```diff
@@ -0,0 +1,12 @@
+import { TemplateRef } from '@angular/core';
+import { DynamicField } from './dynamic-field';
+
+export class CustomField<T = any> extends DynamicField<T> {
```

Comment:
I'd prefer TemplateField. What's the opinion of @lincrosenbach @lvoliveira?

Reply:
I think this field is composed of a formControl + a template and is motivated to give the programmer a way to add a custom field to the form, right?

Reply:
I didn't like TemplateField either, looks like it is a Template of a field

Reply:
Perhaps, we should find other examples with custom/template implementation in shared libraries and follow the most used case.

Reply:
Custom and Template are both clear to me what it is.

Reply:
ok then, let it be